api/AudioBufferSourceNode.json

```diff
@@ -203,7 +203,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/AudioBufferSourceNode/loop",
           "support": {
             "chrome": {
-              "version_added": "14"
+              "version_added": "15"
```

Comment:
This one was confirmed manually via my own Chromium VMs -- `loop` was `false` in Chrome 15 but `undefined` in Chrome 14.  I think this may have been one of my own adjustments, as we don't have any Chrome data in the collector for anything below Chrome 26.

Reply:
OK. Maybe it was `looping` before, but it's not worth the effort to find out really.